nox/sessions.py

```diff
@@ -184,6 +185,13 @@ def create_tmp(self) -> str:
         self.env["TMPDIR"] = tmpdir
         return tmpdir
 
+    @property
+    def cache_dir(self) -> pathlib.Path:
+        """Create and return a 'shared cache' directory to be used across sessions."""
+        path = pathlib.Path(self._runner.envdir).joinpath(".shared")
```

Comment:
Because we want access to the root `.nox` directory rather than the `.nox/<session_name>` directory that most other session properties want access to, we'll need the `envdir`'s parent here:

```python
path = pathlib.Path(self._runner.envdir).parent.joinpath(".shared")
path.mkdir(exist_ok=True)
return path
```